packages/@lwc/engine-core/src/framework/utils.ts

```diff
@@ -48,9 +60,10 @@ export function addCallbackToNextTick(callback: Callback) {
 
 export function guid(): string {
     function s4() {
-        return Math.floor((1 + Math.random()) * 0x10000)
-            .toString(16)
-            .substring(1);
+        return StringSubstring.call(
+            NumberToString.call(Math.floor((1 + Math.random()) * 0x10000), 16),
+            1
+        );
```

Comment:
Needed the actual `Number.prototype.toString` here or else TypeScript doesn't understand the `16` param.